fixtures/env.test-agents

```diff
@@ -0,0 +1,29 @@
+# Agent environment var overrides
+
+KATHY_CHAT_MESSAGES="Chat message 1,Chat message 2,Chat message 3"
+KATHY_INTERVAL=999
+KATHY_ENABLED=true
+
+PROCESSOR_ALLOWED=0x1111111111111111111111111111111111111111111111111111111111111111,0x1111111111111111111111111111111111111111111111111111111111111112,0x1111111111111111111111111111111111111111111111111111111111111113
```

Comment:
@arnaud036 thoughts on using comma separated string for loading list through env vars? I know we currently use `REPLICA_{number}_NAME` for the replica list but it gets a bit crazy for agent config lists. Thoughts on?
- Switching loading replicas to comma-separated list + using comma list for agent configs
- Using indexed entries like replica for agent configs
- Using indexed entries for replica list BUT comma separated list of agent configs